pom.xml

```diff
@@ -51,7 +51,7 @@
     <vertx.version>4.3.3</vertx.version>
 
     <!-- Security update overrides -->
-    <snakeyaml.version>1.31</snakeyaml.version>
```

Comment:
There is a Vert.x 4.3.4 release now that's supposed to fix the SnakeYAML issue so we might be able to remove this override. Bump Vert.x version and see.